04/passenger-service/src/main/java/cz/muni/fi/airportmanager/passengerservice/model/examples/Examples.java

```diff
@@ -0,0 +1,72 @@
+package cz.muni.fi.airportmanager.passengerservice.model.examples;
+
+public class Examples {
+    private Examples() {
+    }
+
+    public static final String VALID_PASSENGER = """
```

Comment:
Are these examples used somewhere? If not, then don't put them into code but rather into README or different example file.

Reply:
Yes in open api specification